### ISSUE TITLE
Remove your soundcloud api key so people cannot abuse it!

### ISSUE THREAD

**Issue by mainrs** [kulpae/cloudruby#21]

**Comment by kulpae** [comment]
Yes, at some point the key was worth the bits it was stored on, and it stopped working. Sadly there is no legal way to use the SoundCloud API for external playback or downloads anymore. But yeah, the key is not in the main branch anymore, only part of the git history :D
